generate-pdf.js

```diff
@@ -73,16 +84,21 @@ async function getUrlPdfCli() {
       type: "boolean",
       describe: "if present, contents of the PDF will be written to standard out"
     })
-    .option("optionOverrides", {
+    .option("optionOverridesFile", {
       alias: "o",
-      describe: "json representing overrides to default pdf print parameters"
+      describe: "path to json file contianing overrides to default pdf print parameters"
     })
     .check(validatePdfCliArgs)
     .help().argv;
 
   try {
     const browser = await puppeteer.launch();
-    var pdfContents = await getUrlPdf(browser, argv.url, argv.writePath, argv.optionOverrides)
+    var pdfContents = await getUrlPdf(
+      browser,
+      argv.url,
+      argv.writePath,
+      argv.optionOverrides && readJsonFile(argv.optionOverrides)
```

Comment:
What is `readJsonFile`? (`parseJsonFile`?)

Reply:
it should be parseJsonFile. Missed this because there is no unit/integration test for the cli function itself